templates/kotlin/src/main/kotlin/io/appwrite/models/Payload.kt.twig

```diff
@@ -0,0 +1,67 @@
+package io.appwrite.models
+
+import io.appwrite.extensions.gson
+import java.io.File
+import java.net.URLConnection
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class Payload private constructor() {
+
+    lateinit var path: String
+    lateinit var filename: String
```

Comment:
I feel like I am missing something. How does filename get to API?

Reply:
[here](https://github.com/appwrite/sdk-generator/blob/4d2f3eaa9fcad9cc73478276ace2cd98392bbf13/templates/kotlin/src/main/kotlin/io/appwrite/Client.kt.twig#L349) for example